Game/Minigames/ClickAtTheRightTime/ClickAtTheRightTime.gd

```diff
@@ -9,72 +10,30 @@ var cursorSpeed = 1.0
 var timer = 10.0
 var timeLeft = timer
 var difficulty = 5.0
-var redPart = 0.1
 var freeze = false
 var goldenZoneVisible = false
 var isBlindFoldedVersion = false
 
-var hasAdvancedPerk = false
+var hasAdvancedPerk = true
+var hardStruggleEnabled = true
 var perfectStreak = 0
 var howMuchForPerfect = 0.25
 
-var difficultySettings = [
```

Comment:
I kinda liked the idea of handcrafted difficulties, makes balancing easier. Maybe keep these but if the difficulty is above 5 use the formulas instead?